webview/src/plots/hooks/useGetPlot.ts

```diff
@@ -10,24 +10,15 @@ import { PlainObject, VisualizationSpec } from 'react-vega'
 import { plotDataStore } from '../components/plotDataStore'
 import { PlotsState } from '../store'
 
-const getStoreSection = (section: Section) => {
-  switch (section) {
-    case Section.CHECKPOINT_PLOTS:
-      return 'checkpoint'
-    case Section.TEMPLATE_PLOTS:
-      return 'template'
-    default:
-      return 'custom'
-  }
-}
+const getStoreSection = (section: Section) =>
+  section === Section.TEMPLATE_PLOTS ? 'template' : 'custom'
 
 export const useGetPlot = (
   section: Section,
   id: string,
   spec?: VisualizationSpec
 ) => {
-  const isPlotWithSpec =
-    section === Section.CHECKPOINT_PLOTS || section === Section.CUSTOM_PLOTS
+  const isPlotWithSpec = section === Section.CUSTOM_PLOTS
```

Comment:
This used to be higher and used for the check on line 13 as well. I think most of this file can be reverted to what it used to before duplication of checkpoints and custom (https://github.com/iterative/vscode-dvc/commit/165108829bdfaf35d5c8df39fc5efdcfa49c9772#diff-164992d57e219d2123695397d166b9b166a6aff3833b781b4241e22b5078e0d7)